sdk/keyvault/azure-security-keyvault-common/inc/azure/keyvault/common/internal/keyvault_pipeline.hpp

```diff
@@ -148,7 +148,7 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Common { n
     {
       auto request = CreateRequest(method, path);
       // Use the core pipeline directly to avoid checking the response code.
-      return m_pipeline.Send(context, request);
+      return m_pipeline.Send(request, context);
```

Comment:
Anything else in keyvault need changing, @vhvb1989? Any implementation detail or APIs where context is first?

Reply:
All API's uses this common function to send a request. :)

Reply:
I meant other than pipeline.Send. Are there any other KeyVault APIs that I need to update?